appcues/src/main/java/com/appcues/ui/extensions/ImageLoaderExt.kt

```diff
@@ -0,0 +1,68 @@
+package com.appcues.ui.extensions
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.drawable.BitmapDrawable
+import android.os.Build.VERSION
+import android.os.Build.VERSION_CODES
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.graphics.painter.BitmapPainter
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import coil.ImageLoader
+import coil.decode.GifDecoder
+import coil.decode.ImageDecoderDecoder
+import coil.decode.SvgDecoder
+import coil.request.ImageRequest
+import coil.size.Scale
+import coil.size.Size
+import com.appcues.data.model.styling.ComponentContentMode
+import com.appcues.data.model.styling.ComponentContentMode.FILL
+import com.appcues.data.model.styling.ComponentContentMode.FIT
+import com.appcues.data.model.styling.ComponentSize
+
+internal fun Context.getImageLoader(): ImageLoader {
```

Comment:
added image loader related calls here so we can reuse both for ImagePrimitive and for backgroundImage property.